src/category_theory/preadditive/additive_functor.lean

```diff
@@ -128,6 +130,24 @@ lemma additive_of_preserves_binary_biproducts [has_binary_biproducts C] [preserv
     ← biprod.map_biprod_hom_desc, category.assoc, iso.inv_hom_id_assoc, F.map_id,
     biprod.add_eq_lift_id_desc] }
 
+section
+local attribute [instance] preserves_binary_biproducts_of_preserves_binary_products
+
+lemma additive_of_preserves_finite_limits [has_zero_object C] [has_zero_object D]
+  [has_binary_biproducts C] [preserves_finite_limits F] : additive F :=
```

Comment:
Shouldn't `preserves_binary_products` be enough?

Reply:
Yes. I removed the lemmas completely.